### DISCUSSION TITLE
How to input a token when using Mongoose as an HTTP client for authenticated website access?

### DISCUSSION THREAD

**Discussion by Yhee2** [cesanta/mongoose#2296] [Ask questions, get answers] [answered]
Hello, sorry to bother, but if I use Mongoose as a client to access an authenticated website via HTTP, how can I input the token? I would greatly appreciate it.

I attempted to add the token at this location, but it is incorrect.
mg_printf(c,
"%s %s HTTP/1.0\r\n"
"Host: %.*s\r\n"
"Content-Type: application/json\r\n"
"Content-Length: %d\r\n"
"token: xxxxxxxxxxxxxxxxxxxxxxxx"
"\r\n",
s_post_data ? "POST" : "GET", mg_url_uri(s_url), (int) host.len,
host.ptr, content_length);

**Comment by scaprile** [comment, marked as answer]
Please do not crosspost nor thread-hijack nor necropost.
https://github.com/cesanta/mongoose/discussions/2227#discussioncomment-6401531

**Reply by Yhee2** [reply]
I'm truly sorry, my apologies. I will delete this question.
